torch/csrc/distributed/rpc/init.cpp

```diff
@@ -6,6 +6,7 @@
 #include <torch/csrc/distributed/rpc/python_rpc_handler.h>
 #include <torch/csrc/distributed/rpc/rpc_agent.h>
 #include <torch/csrc/distributed/rpc/rref_context.h>
+#include <torch/csrc/distributed/rpc/tensorpipe_agent.h>
```

Comment:
Not requesting for change in this PR, we probably should guard this and the `process_group_agent.h`  above (and other related parts) with `ifdef`. Added #37782 to track.